src/stores/pacbioRunCreate.js

```diff
@@ -249,12 +283,10 @@ export const usePacbioRunCreateStore = defineStore('pacbioRunCreate', {
         id,
         // This is long but we want to include pool data
```

Comment:
May be good to update the documentation as we are including library data as well